frappe/templates/includes/login/login.js

```diff
@@ -287,11 +287,11 @@ var continue_otp_app = function(setup, qrcode){
 	var qrcode_div = $('<div class="text-muted" style="padding-bottom: 15px;"></div>');
 
 	if (setup){
-		direction = $('<div>').attr('id','qr_info').text('Enter Code displayed in OTP App.');
+		direction = $('<div>').attr('id','qr_info').text(_('Enter Code displayed in OTP App.'));
```

Comment:
Please use `__` (double underscores) instead of single underscore for translating strings in `.js` files.